parlai/core/worlds.py

```diff
@@ -525,6 +537,10 @@ def get_agents(self):
         """Return the agents in the *current* subworld."""
         return self.worlds[self.world_idx].get_agents()
 
+    def get_task_agent(self):
+        """Return the task agent in the *current* subworld."""
+        return self.worlds[self.world_idx].get_task_agent()
```

Comment:
This is a really funny definition, because there could be many task agents... Maybe this one should just raise a RuntimeError? If we do that, do unit tests fail?

Do we have unit tests covering multitask training? I don't think we do. We need those asap.

Reply:
i was trying to match the functionality of the function before, i.e. `def get_agents(self)`. I agree that this is a weird definition for a teacher where there are multiple `task_agents`

we actually test multitasking in two places: `test_train_model` (tests the multitask metrics) and `test_pytorch_data_teacher` (tests multitasking with that). I can try seeing if raising a `RuntimeError` will break those tests